core/versions_2.0.md

```diff
@@ -0,0 +1,71 @@
+# Workflow versioning 2.0
+
+This proposal introduces two key changes to what is called `getVersion` in the existing SDKs, but will likely be renamed in subsequent ones (more on naming later). The key changes are:
+
+* Introduce a different API for versioning that does not involve a version range, but instead simply returns a boolean indicating if a named change has happened or not. Ex: `hasChange("change-id")`. TBD if we would eliminate the range-based API, or leave it for advanced users.
```

Comment:
I think the `changed("change-x")` call is a bit confusing, b/c it looks like "Has 'change-x' changed?".

`hasChange` read a bit more natural to me, but I would also throw `supports("feature-x")` into the ring

Reply:
+1 `hasChange`, but -1 `supports`, the latter one is too ambiguous, I think.

Reply:
Another option is `usePatch('patch-x')`, I find it conveys the concept better, because it's not about if you have the change or patch but more about if you should use it for a specific branch (yes, if you're not replaying or replaying code that used this patch before).